### PR TITLE
Upate URIs for some LBZ vocabs

### LINKED ISSUE #2: Update perma URI redirects
- [x] http://purl.org/lobid/rpb -> https://rpb.lobid.org/subjects
- [x] https://w3id.org/lobid/rpb2 -> https://skohub.io/hbz/lbz-vocabs/heads/main/w3id.org/lobid/rpb2
- [x] https://w3id.org/lobid/rpb-fachgebiete/ -> https://skohub.io/hbz/lbz-vocabs/heads/main/w3id.org/lobid/rpb-fachgebiete/

Nothing to do for https://rpb.lobid.org/spatial

### PR DESCRIPTION
**Pull request by acka47**
Should partly resolve https://github.com/hbz/lbz-vocabs/issues/2

### CODE REVIEW

**Comment by acka47** [comment]
@Phu2 @dr0i  could one of you please have a look before I make a PR against https://github.com/perma-id/w3id.org ? I just want to make sure that redirects will work directly.

File: lobid/.htaccess [hunk 1]
```diff
@@ -1,4 +1,5 @@
 
 Options +FollowSymLinks
 RewriteEngine on
-RewriteRule ^rpb2$ https://raw.githubusercontent.com/lobid/vocabs/master/rpb/rpb2.ttl [R=302,L]
\ No newline at end of file
+RewriteRule ^rpb2$ https://skohub.io/hbz/lbz-vocabs/heads/main/w3id.org/lobid/rpb2 [R=302,L]
```
Comment (Phu2) [inline lobid/.htaccess]: This rule/regex doesn't match paths with fragments like rpb2#n120.
https://w3id.org/lobid/rpb2#n120 > https://skohub.io/hbz/lbz-vocabs/heads/main/w3id.org/lobid/rpb2.html#n120
I think it needs to be:
`RewriteRule ^rpb2(.*) https://skohub.io/hbz/lbz-vocabs/heads/main/w3id.org/lobid/rpb2$1 [R=302,L]`

File: lobid/.htaccess [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
+RewriteRule ^rpb-fachgebiete/(.*) https://skohub.io/hbz/lbz-vocabs/heads/main/w3id.org/lobid/rpb-fachgebiete/ [R=302,L]
```
Comment (Phu2) [inline lobid/.htaccess]: The `$1` back reference is missing in the substituion part of the rule.
Should be
`RewriteRule ^rpb-fachgebiete/(.*) https://skohub.io/hbz/lbz-vocabs/heads/main/w3id.org/lobid/rpb-fachgebiete/$1 [R=302,L]`

File: lobid/.htaccess (hunk 1 above)
Comment (acka47) [reply lobid/.htaccess]: I was thinking about this myself. But the hash and everything after it aren't ~~submitted to~~ handled by the server anyway but only client-side. So, it should not make any difference. Also, it currently looks vey similar:

https://github.com/perma-id/w3id.org/blob/7c72f2257097810490ec2860aa0d5f08666a7fd2/lobid/.htaccess#L4

File: lobid/.htaccess (hunk 2 above)
Comment (acka47) [reply lobid/.htaccess]: Thanks, will update.

File: lobid/.htaccess (hunk 1 above)
Comment (acka47) [reply lobid/.htaccess]: Prbably my thinking was wrong. Updated it Thanks!

**Pull request merged**